src/actions.js

```diff
@@ -501,3 +516,93 @@ export const walletStateReady = () => ({
 export const onWalletReload = () => ({
   type: types.WALLET_RELOADING,
 });
+
+// Push notification actions
+
+/**
+ * @param {{deviceId: string, settings: { enabled, showAmountEnabled }, hasBeenEnabled: boolean, enabledAt: number}} payload
+ */
+export const pushInit = (payload) => ({
```

Comment:
Remove (Explained in next comments)

Reply:
I keep this initialization because the push init on saga use this action to update the redux store.